src/v/archival/tests/ntp_archiver_test.cc

```diff
@@ -1701,21 +1701,32 @@ static void test_manifest_spillover_impl(
     }).get();
 
     // Generate new manifest based on data layout on disk
-    std::vector<cloud_storage::segment_meta> all_segments;
+    vlog(test_log.debug, "stm add segments");
```

Comment:
should a commit message say: `adding a segment` instead of `adding a second` ?

Reply:
Indeed. Reworded the commit message.